pkg/service/service.go

```diff
@@ -1142,3 +1240,23 @@ func isWildcardAddr(frontend lb.L3n4AddrID) bool {
 	}
 	return net.IPv4zero.Equal(frontend.IP)
 }
+
+func segregateBackends(backends []lb.Backend) (map[string]lb.BackendID, []lb.BackendID) {
```

Comment:
worth a comment on this function.

Reply:
This function isn't exported. The function has detailed comments to explain what it does. But to make it more explicit, I used named return parameters.